src/SeawaterPolynomials.jl

```diff
@@ -84,10 +84,10 @@ This function aliases `total_density`.
 """
     ρ′(Θ, Sᴬ, Z, equation_of_state)
 
-Returns the total density of a seawater parcel with conservative tempertuare `Θ`, absolute
+Returns the density anomaly of a seawater parcel with conservative tempertuare `Θ`, absolute
```

Comment:
```suggestion
Return the density anomaly of a seawater parcel with conservative tempertuare `Θ`, absolute
```

Reply:
Thanks for catching this spelling error!!